views/lib/error_content.hbs

```diff
@@ -4,12 +4,17 @@
     <p class="error-header-text">
         {{#if error.statusCode}}
             {{castStatusCodeToString error.statusCode}}
+        {{else if title}}
+            {{title}}
         {{else}}
             Da ist wohl etwas schief gelaufen!
         {{/if}}
     </p>
-
-    <button class="btn btn-primary" onclick="window.history.back();"><i class="fa fa-arrow-left"></i> zurück</button>
+    {{#if reload}}
+        <button class="btn btn-primary" onclick="window.location.href=window.location.href">Neu laden</button>
```

Comment:
here we should only re-do the same request with the updated csrf token instead of this reload/back logic, see max comment in the ticket, at least: we have to keep all form data on browser.back but must update the csrf in the form